include/s3select.h

```diff
@@ -1191,7 +1188,6 @@ class csv_object : public base_s3object
   bool m_aggr_flow = false; //TODO once per query
   bool m_is_to_aggregate;
   bool m_skip_last_line;
-  size_t m_stream_length;
```

Comment:
why did you remove that?

Reply:
it appears not in use, (using other variable to mark end of-stream)